supabase/migrations/20230924193504_label_changes.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS label_changes (
+    id serial PRIMARY KEY,
+    created timestamptz NOT NULL,
+    updated timestamptz NOT NULL,
+    username text NOT NULL,
+    repository text NOT NULL,
+    label_from text NOT NULL,
+    label_to text NOT NULL,
+    approved boolean NOT NULL
```

Comment:
```suggestion
    authorized boolean NOT NULL
```
@seprintour lets change everything to `authorized` instead of `approved` and `authorize` instead of `approve`

Reply:
Makes sense, i've edited that